docs/ALGOLIA-SEARCH.md

```diff
@@ -46,6 +46,32 @@ NexT provides search plugin for index your hexo website content. To use this fea
 
     ![](http://theme-next.iissnan.com/uploads/algolia/algolia-step-4.png)
 
+1. Change dir to NexT directory, and install module to `source/lib` directory.
+
+    ```
+    $ cd themes/next
+    $ git clone https://github.com/theme-next/theme-next-algolia-instant-search source/lib/algolia-instant-search
+    ```
+
+    If you want to use the CDN instead of clone this repo, then need to **set vendors** in NexT `_config.yml` file and **add** custom code in `~/themes/next/source/css/_custom/custom.styl` or `~/source/_data/styles.styl`
+
+    ```yml
+    vendors:
+      ...
+      # Internal version: 1
+      # https://www.algolia.com
+      algolia_instant_js: https://cdn.jsdelivr.net/npm/instantsearch.js@2.4.1/dist/instantsearch.js
+      algolia_instant_css: https://cdn.jsdelivr.net/npm/instantsearch.js@2.4.1/dist/instantsearch.min.css
+      ...
+    ```
+
+    ```
+    // Custom styles.
```

Comment:
```
.ais-search-box--magnifier svg {
    display: none !important;
}
```

It seems they are all working. But I don't know how to choose one.

Reply:
Confirmed. The PR is right.

Reply:
@sli1989 so, maybe u can add this to [lib plugin](https://github.com/theme-next/theme-next-algolia-instant-search)?

Reply:
Oh no, wait. This need just add to styles in NexT i think.

Reply:
ok, working on it.

Reply:
emm, i will try it.